service/src/main/kotlin/fi/espoo/evaka/application/ApplicationForm.kt

```diff
@@ -523,4 +523,15 @@ data class ServiceNeedOption(
     val nameSv: String,
     val nameEn: String,
     val validPlacementType: PlacementType?
-)
+) {
+    companion object {
+        fun of(option: fi.espoo.evaka.serviceneed.ServiceNeedOption) =
+            ServiceNeedOption(
+                option.id,
+                option.nameFi,
+                option.nameSv,
+                option.nameEn,
+                option.validPlacementType
+            )
+    }
+}
```

Comment:
Since this function is only used in tests I think a function defined in a test file would be a more appropriate place for this.

Reply:
Updated!